src/util/heartbeat.ts

```diff
@@ -8,12 +6,16 @@ export interface Heartbeat {
   getSeconds: () => number
 }
 
+interface WritableLike {
+  write: (chunk: any) => void
```

Comment:
This should be `chunk: string`, since that's a subset of what `Writable` takes, and is the most specific thing the hearbeat needs.